iamy/aws.go

```diff
@@ -434,6 +435,14 @@ func (a *AwsFetcher) isSkippableManagedResource(cfnType CfnResourceType, resourc
 		}
 	}
 
+	if len(a.SkipPathPrefixes) > 0 {
+		for _, path := range a.SkipPathPrefixes {
```

Comment:
Can we use this to replace the code below that checks for `aws-service-role`? Given that's doing the same thing

Reply:
I didn't want to do that primary for clarity about why things are skipped.  Theoretically we could replace all of the `AWSServiceRole` skipping stuff with a more general purpose --skip-name-pattern or something like that.    I'll tackle that in the next PR.